singularity-eos/eos/eos_SpinerEOSDependsRhoSie.hpp

```diff
@@ -0,0 +1,731 @@
+#ifndef _EOS_SPINEREOSDEPENDSRHOSIE_HPP_
+#define _EOS_SPINEREOSDEPENDSRHOSIE_HPP_
```

Comment:
These include guards should be the full relative path from the top level of the repository, separated by underscores instead of forward slashes. So if you take my above suggestion for names, this would be
```suggestion
#ifndef _EOS_EOS_SPINER_DEPENDS_RHO_SIE_HPP_
#define _EOS_EOS_SPINER_DEPENDS_RHO_SIE_HPP_
```